packages/ComponentLibrary/src/components/Table/TabNavigation/TabContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import TabContent from './TabContent';
+import type { TabProps } from '../../../../../storybook/src/stories/Components/Table/types';
```

Comment:
hmmm this doesn't look good.
As far as I can understand, the `TabProps` type should be part of the ComponentLibrary